hudi-common/src/main/java/org/apache/hudi/common/util/ClusteringUtils.java

```diff
@@ -215,6 +215,12 @@ public static HoodieClusteringPlan createClusteringPlan(String strategyClassName
   }
 
   public static List<HoodieInstant> getPendingClusteringInstantTimes(HoodieTableMetaClient metaClient) {
-    return metaClient.getActiveTimeline().filterPendingReplaceTimeline().getInstants().collect(Collectors.toList());
+    return metaClient.getActiveTimeline().filterPendingReplaceTimeline().getInstants()
+            .filter(instant -> isPendingClusteringInstant(metaClient, instant))
```

Comment:
this is unrelated to current change. But important bug fix. We were considering all replace commits as 'Clustering' commits before.

Reply:
thanks for the fix, I was planning to ask the same which I detected while investigating the issue. good you fixed it :)

Reply:
CC @codope